docs/API/V1/server_config.md

```diff
@@ -223,7 +230,7 @@ For example, the following are examples of valid banners:
 
 This server configuration setting allows a server administrator to control
 the operating state of the server remotely. As for
-[`server-banner`](#server-banner), the value is a JSON object, and any JSON
+[`server-banner`](#server-banner), the value is a JSON object and any JSON
 fields passed in to the server will be returned to a client, however the
 following fields have special meaning:
```

Comment:
Um...God I hate to say this, but...that comma was one that we needed.  🤣  The subsequent `and` is connecting two full sentences, so it should be preceded by a comma.  🙁

Now, if you want to really dive in, `however` is not a conjunction, and so when it joins two sentences (as it does here), it should be preceded by a semicolon, not a comma.  😂

Reply:
Nope, nevermind, I was never here, move along.